packages/flutter_tools/lib/src/commands/start.dart

```diff
@@ -109,10 +117,12 @@ class StartCommand extends StartCommandBase {
 }
 
 Future<int> startApp(
+  BuildConfiguration config,
```

Comment:
Done. I also changed it to pass the List of BuildConfigurations, since this command is platform agnostic.